### DISCUSSION TITLE
What To Delete?

### DISCUSSION THREAD

**Discussion by djphazer** [djphazer/O_C-Phazerville#42] [Ideas]
Many of us love to get lost in the sauce and enjoy exploring the huge toolbox that is O_C. But one of the glaring downsides to this module/firmware is option paralysis - there are simply **_too many things_** to play with and get sidetracked by. Too many choices and configuration settings can kill the creative flow.

I've seen plenty of folks express distaste in the esoteric nature of the stock apps, while the relative simplicity of Hemispheres seems more accessible. If you hate menu-diving, this is probably not the module for you, but as I continue to add to the complexity, I'm curious about what people view as unnecessary bloat.

Just as a thought experiment, what would you eliminate to keep things simple?

**Comment by xmacex** [comment]
Nothing ;)

**Comment by yratof** [comment]
Metronome

**Comment by yratof** [comment]
Binary clock

**Comment by 5-lagu** [comment]
I know that this will be higly subjective, but I rarely or never use: Palimpset, DrumMap, Lofi echo, GAted VCA, Metronome, MIDI*, Runglebook, SchmittTr, TL Neuron, Trending, "A"SR. I also very rarely use the (Vector-)envelope-generators. I think the new DualTM is so all-encompassing that Enigma Jr is kind of obsolete too. Just my two-ish cents.
